webhook/webhook.go

```diff
@@ -144,6 +144,7 @@ func New(
 	logger := logging.FromContext(ctx)
 
 	if opts.StatsReporter == nil {
+		RegisterMetrics()
```

Comment:
If there are multiple webhooks, this function might end up with being called multiple times and that won't work (which failed the unit test).